AFQ/_fixes.py

```diff
@@ -76,11 +76,12 @@ def _verbose_generate_tractogram(self):
             # move to the next streamline if only the seed position
             # and not return all
             len_sl = len(streamline)
-            if len_sl > self.min_length:
-                if self.save_seeds:
-                    yield streamline, s
-                else:
-                    yield streamline
+            if len_sl >= self.min_length:
```

Comment:
Is there any kind of test we can add here, to make sure this is doing the right thing now?